File: relationship-manager/product-catalog-service/src/main/java/com/bank/rm/product/ProductCatalogApplication.java
package com.bank.rm.product;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.product", "com.bank.rm.common"})
public class ProductCatalogApplication {
    public static void main(String[] args) { SpringApplication.run(ProductCatalogApplication.class, args); }
}

File: relationship-manager/product-catalog-service/src/main/kotlin/com/bank/rm/product/ProductCatalogApplication.kt
package com.bank.rm.product

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.product", "com.bank.rm.common"])
class ProductCatalogApplication

fun main(args: Array<String>) {
    runApplication<ProductCatalogApplication>(*args)
}
